CHANGELOG.md

```diff
@@ -38,6 +43,16 @@ Note, since the pipeline is now using Nextflow DSL2, each process will be run wi
 | `samtools`              | 1.10        | 1.15.1      |
 | `picard`                | 2.23.1      | 2.27.4      |
 | `preseq`                | 2.0.3       | 3.1.2       |
+| `r-base`                | 3.6.1       | 4.0.3       |
+| `r-tidyr`               | -           | -           |
+| `r-lattice`             | -           | -           |
+| `r-xfun`                | -           | -           |
+| `bioconductor-vsn`      | -           | -           |
+| `trim-galore`           | 0.6.5       | 0.6.7       |
```

Comment:
Reordered to group them by updated, new and removed; and added `r-optparse` from [here](https://github.com/nf-core/chipseq/blob/6601012f0d022164e66379bd1d652580a989f406/modules/local/macs2_consensus.nf#L8)
```suggestion
| `r-base`                | 3.6.1       | 4.0.3       |
| `trim-galore`           | 0.6.5       | 0.6.7       |
| `r-optparse`            | -           | 1.7.1       |
| `r-tidyr`               | -           | -           |
| `r-lattice`             | -           | -           |
| `r-xfun`                | -           | -           |
| `bioconductor-vsn`      | -           | -           |
```